File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    answer = 0
    dots.sort()
        
    x1, y1 = dots[0][0], dots[0][1]
    x2, y2 = dots[-1][0], dots[-1][1]
    
    answer = (x2 - x1) * (y2 - y1)
    
    return answer

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
import java.util.*;

class Solution {
    public int solution(int[][] dots) {
        Arrays.sort(dots, new Comparator<int[]>() {
            @Override
            public int compare(int[] o1, int[] o2) {
                return o1[0] != o2[0] ? o1[0] - o2[0] : o1[1] - o2[1];
            }
        });

        int width = dots[3][0] - dots[0][0];
        int height = dots[3][1] - dots[0][1];

        int answer = width * height;
        return answer;
    }
}